gcp/api/server.py

```diff
@@ -131,7 +132,7 @@ def do_query(query, context, include_details=True):
       context.abort(grpc.StatusCode.INVALID_ARGUMENT, 'Invalid Package URL.')
       return None
 
-  to_response = lambda b: bug_to_response(b, include_details)
+  to_response = lambda b: bug_to_response(b, include_details)  # pylint: disable=unnecessary-lambda-assignment
```

Comment:
Let's just add this to https://github.com/google/osv.dev/blob/f660fa6a7dc02337f1944cd276181a3bf4abd0e1/.pylintrc#L10

Reply:
I reckon this is fine. Up to you to make the judgement call if you think there's value to keeping this check.

Reply:
This is now a redundant conversation thanks to @another-rex and 1ab0afb